Cargo.toml

```diff
@@ -11,3 +11,4 @@ license = "MIT"
 [dependencies]
 byteorder = "0.5"
 crc = "1.3.0"
+enum_primitive = "*"
```

Comment:
I would prefer this dependency to have a fixed version so it would have less chance of breaking the code if the api changes. Also I think cargo requires crates to have fixed version dependencies before it can be uploaded to crates.io.